funnel/views/session.py

```diff
@@ -198,6 +198,12 @@ def delete(self):
         else:
             self.obj.make_unscheduled()
         db.session.commit()
+        if self.obj.project.features.schedule_publish_alert():
+            return jsonify(
+                status=True,
+                modal_url=modal_url,
+                msg='This project will not be listed as it has no sessions in the schedule',
```

Comment:
`msg=_("This project will not be listed as it has no sessions in the schedule"),`